src/test/java/seedu/uninurse/logic/commands/FindCommandTest.java

```diff
@@ -54,13 +54,13 @@ public void equals() {
     }
 
     @Test
-    public void execute_zeroKeywords_noPersonFound() {
-        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 0);
+    public void execute_zeroKeywords_allPersonFound() {
+        String expectedMessage = String.format(MESSAGE_PERSONS_LISTED_OVERVIEW, 7);
```

Comment:
Maybe use `model#getFilteredPersonList()#size()` instead of `7`?

Reply:
Fixed